### PR TITLE
Added instructions for Angular 7 and Angular Universal Starterkit

### PR DESCRIPTION
**Pull request by Ballpin**
Just yesterday i noticed that Angular 8 was released. These instructions have not been used with version 8.

But for version 7 it has been tried.

Related to issue https://github.com/tailwindcss/docs/issues/229

### CODE REVIEW

**Comment by genu** [comment]
There needs some clarity around what do with exiting configuration. For example, are we to remove all existing options? or just append to the options.

File: source/docs/installation.blade.md [hunk 1]
```diff
@@ -275,6 +275,121 @@ module.exports = function(defaults) {
...
+
+    to 
+    
+    ```js
+    {
+        // ...
+        "build": {
+            "builder": "@angular-builders/custom-webpack:browser",
+            "options": {
+                "customWebpackConfig": {
```
Comment (genu) [inline source/docs/installation.blade.md]: Should the previous options be removed or left here?

File: source/docs/installation.blade.md [hunk 2]
```diff
@@ -275,6 +275,121 @@ module.exports = function(defaults) {
...
+
+    ```js
+    {
+        // ...
+        "build": {
+            "builder": "@angular-devkit/build-angular:browser",
+        }
+        //...
+        "serve": {
+            "builder": "@angular-devkit/build-angular:dev-server",
```
Comment (genu) [inline source/docs/installation.blade.md]: In newer version, this should be changed to `"builder": "@angular-builders/custom-webpack:dev-server"` in order to work I believe

File: source/docs/installation.blade.md [hunk 3]
```diff
@@ -275,6 +275,121 @@ module.exports = function(defaults) {
...
+        }
+        //...
+        "server": {
```
Comment (genu) [inline source/docs/installation.blade.md]: Angular cli 8 doesn't have a "server" block, so I think this can go away.

Comment (Ballpin) [reply source/docs/installation.blade.md]: The version i used is from Angular Universal Starter Kit (https://github.com/angular/universal-starter).  [See the line here](https://github.com/angular/universal-starter/blob/3196db10ee627110cb6cd1414b610e0253e2af3b/angular.json#L108). I belive i mentioned it in the begining.

File: source/docs/installation.blade.md (hunk 2 above)
Comment (Ballpin) [reply source/docs/installation.blade.md]: Yes, you are correct i missed it. I will rewrite that part.

**Comment by chadams** [comment]
step 4 is vague

**Comment by HonkingGoose** [comment]
Angular 7 is no longer supported: https://angular.io/guide/releases#support-policy-and-schedule

From the official Angular docs:
> Angular versions ^4.0.0, ^5.0.0, ^6.0.0 and ^7.0.0 are no longer under support.

Angular support is as follows:
^10.0.0 | Active | Jun 24, 2020 | Dec 24, 2020 | Dec 24, 2021
-- | -- | -- | -- | --
^9.0.0 | Active | Feb 06, 2020 | Aug 06, 2020 | Aug 06, 2021
^8.0.0 | LTS | May 28, 2019 | Nov 28, 2019 | Nov 28, 2020

Note that Angular 8 is soon to be _EOL_ too after Nov 28 2020.

**Comment by royvanv** [comment]
See https://github.com/tailwindlabs/tailwindcss-setup-examples/pull/35

**Comment by simonswiss** [comment]
Hey, doing some "end of the year" cleaning of issues / PRs.

We've now released a few [integration guides](https://tailwindcss.com/docs/installation#integration-guides) on the new website.

There is currently no Angular guide, but@Ballpin would you be interested in trying to do draft an Angular guide that matches the content of the existing guides?

**Comment by royvanv** [comment]
> Hey, doing some "end of the year" cleaning of issues / PRs.
> 
> We've now released a few [integration guides](https://tailwindcss.com/docs/installation#integration-guides) on the new website.
> 
> There is currently no Angular guide, but@Ballpin would you be interested in trying to do draft an Angular guide that matches the content of the existing guides?

@simonswiss, you missed a space before the @.

Ping @Ballpin

**Comment by vltansky** [comment]
@simonswiss this guide is actually wrong. Adding tailwind this way to Angular will break some libraries (e.g angular/material) cause of `postcss-import`, also won't support directives & functions in component styles. 
We faced these issues in [@ngneat/tailwind](https://github.com/ngneat/tailwind):  https://github.com/ngneat/tailwind/issues/15 , https://github.com/ngneat/tailwind/issues/24, https://github.com/ngneat/tailwind/issues/26
fixed in https://github.com/ngneat/tailwind/pull/30


This PR uses `@ngneat/tailwind` https://github.com/tailwindlabs/tailwindcss.com/pull/639

**Comment by adamwathan** [comment]
Hey! Closing just because this is super out of date since we have rebuilt the site with Next.js since this was originally opened. Sorry about that and thanks either way.

**Pull request closed without merging**